docs/zh_cn/user_guides/4_pretrain_custom_dataset.md

```diff
@@ -1,29 +1,29 @@
-# Tutorial 4: Pretrain with Custom Dataset
+# 教程 4: 使用自定义数据集进行预训练
 
-- [Tutorial 4: Pretrain with Custom Dataset](#tutorial-4-pretrain-with-custom-dataset)
-  - [Train MAE on Custom Dataset](#train-mae-on-custom-dataset)
-    - [Get the path of custom dataset](#get-the-path-of-custom-dataset)
-    - [Choose one config as template](#choose-one-config-as-template)
-    - [Edit the dataset related config](#edit-the-dataset-related-config)
-  - [Train MAE on COCO Dataset](#train-mae-on-coco-dataset)
-  - [Train SimCLR on Custom Dataset](#train-simclr-on-custom-dataset)
-  - [Load Pre-trained Model to Speedup Convergence](#load-pre-trained-model-to-speedup-convergence)
+- [教程 4: 使用自定义数据集进行预训练](#教程-4-使用自定义数据集进行预训练)
+  - [在自定义数据集上使用MAE算法进行预训练](#在自定义数据集上使用mae算法进行预训练)
+    - [第一步：获取自定义数据路径](#第一步获取自定义数据路径)
+    - [第二步：选择一个配置文件作为模板](#第二步选择一个配置文件作为模板)
+    - [第三步：修改数据集相关的配置](#第三步修改数据集相关的配置)
+  - [在COCO数据集上使用MAE算法进行预训练](#在coco数据集上使用mae算法进行预训练)
+  - [在自定义数据集上使用SimCLR算法进行预训练](#在自定义数据集上使用simclr算法进行预训练)
+  - [使用MMSelfSup提供的预训练模型来加速收敛](#使用mmselfsup提供的预训练模型来加速收敛)
 
-In this tutorial, we provide some tips on how to conduct self-supervised learning on your own dataset(without the need of label).
+在本教程中，我们将介绍如何使用自定义数据集(无需标注)进行自监督预训练。
 
-## Train MAE on Custom Dataset
+## 在自定义数据集上使用MAE算法进行预训练
 
-In MMSelfSup, We support the `CustomDataset` from MMClassification(similar to the `ImageFolder` in `torchvision`),  which is able to read the images within the specified folder directly. You only need to prepare the path information of the custom dataset and edit the config.
+在MMSelfSup中, 我们支持用户直接调用MMClassification的`CustomDataset`(类似于`torchvision`的`ImageFolder`), 该数据集能自动的读取给的路径下的图片。你只需要准备你的数据集路径，并修改配置文件，即可轻松使用MMSelfSup进行预训练。
 
-### Get the path of custom dataset
+### 第一步：获取自定义数据路径
 
-It should be like `data/custom_dataset/`
+路径应类似这种形式： `data/custom_dataset/`
 
-### Choose one config as template
+### 第二步：选择一个配置文件作为模板
 
-Here, we would like to use `configs/selfsup/mae/mae_vit-base-p16_8xb512-coslr-400e_in1k.py` as the example. We first copy this config file and rename it as `mae_vit-base-p16_8xb512-coslr-400e_${custom_dataset}.py`.
+在本教程中，我们使用 `configs/selfsup/mae/mae_vit-base-p16_8xb512-coslr-400e_in1k.py`作为一个示例进行讲解。我们首先复制这个配置文件，将新复制的文件命名为`mae_vit-base-p16_8xb512-coslr-400e_${custom_dataset}.py`.
```

Comment:
Better to indicate `${custom_dataset}` means dataset name, like `in1k`.
The same comments about English version